docs/user/tutorials/01-70-aws-redis-instance.md

```diff
@@ -0,0 +1,136 @@
+# AWS Redis Instance Tutorial
+This tutorial shows how to instantiate Redis and connect to it.
+s
+## Simple Example
+
+This example showcases how to instantiate Redis, connect a Pod to it, and send a PING command.
+
+1. Instantiate Redis
+
+> [!NOTE]
+> This action may take 10+ min
+
+```yaml
+apiVersion: cloud-resources.kyma-project.io/v1beta1
+kind: AwsRedisInstance
+metadata:
+  name: awsredisinstance-simple-example
+spec:
+  cacheNodeType: cache.t3.micro
+```
+
+2. Instantiate the redis-cli Pod
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: awsredisinstance-simple-example-probe
+spec:
+  containers:
+  - name: redis-cli
+    image: redis:latest
+    command: ["/bin/sleep"]
+    args: ["999999999999"]
+    env:
+    - name: HOST
+      valueFrom:
+        secretKeyRef:
+          key: host
+          name: simple-example-probe
+    - name: PORT
+      valueFrom:
+        secretKeyRef:
+          key: port
+          name: simple-example-probe
+```
+
+3. Exec into the Pod:
+
+```bash
+kubectl exec -i -t awsredisinstance-simple-example-probe -c redis-cli -- sh -c "clear; (bash || ash || sh)"
+```
+
+4. Exec a PING command:
+
+```bash
+redis-cli -h $HOST -p $PORT
+```
+You should receive `PONG` back from the server.
+
+## Complex Example
+
+This example showcases how to instantiate Redis by using most of the spec fields, connect a Pod to it, and send a PING command.
```

Comment:
That makes a lot of sense! 
Then I'd suggest something like this - making the headlines more meaningful along the way.

> ## Basic Setup
> If you don't need auth or TLS, you can instantiate Redis and connect the Pod with only the required fields:
> (...)

> ## Setup with Auth and TLS
> If you want to instantiate Redis and connect the Pod with auth and TLS, use the following setup:
> (...)

Reply:
Hmm, that way it might sound like examples are only different due to auth and TLS, but in fact, the complex example is also showcasing how to specify Redis version, configuration, and maintenance policy.

Reply:
Oh, I see! How about:

> ## Minimal Setup
> To instantiate Redis and connect the Pod with only the required fields, use the following setup:

> ## Advanced Setup
> To specify advanced features (such as Redis version, configuration, and maintenance policy) and set up auth and TLS, use the following setup:

Reply:
Perfect! I will use that!